CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [3.0.1] - 2018-03-30
```

Comment:
Please avoid versioning the changes in a PR per our [guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md) there are several reasons for this:
- PRs are not a FIFO queue, merge order is not guaranteed
- maintainer may disagree with you (in this case its a patch)
- we date the release not the contribution submitted, there is no SLA on review will always happen same day submitted
- attempt to reduce the number of time needing to rebase